content/docs/04-clusters/01-public-cloud/03-azure/00-architecture.md

```diff
@@ -21,4 +21,51 @@ The following are some architectural highlights of Azure clusters deployed by Pa
 
 - None of the control plane nodes and worker nodes have public IPs attached. The Kubernetes API Server endpoint is accessed through a public load balancer.
 
-![azure_cluster_architecture.png](clusters_azure_architecture_overview-diagram.png)
\ No newline at end of file
+
+![An Azure IaaS architecture diagram](clusters_azure_architecture_iaas-overview.png)
+
+
+
+# Azure AKS Architecture
+
+The integration between Palette and Azure AKS unlocks the following capabilities.
+
+- Palette platform enables containerized applications' effortless deployment and management with fully managed AKS. 
+
+
+- Palette provides the you with a with serverless Kubernetes experience, an integrated continuous integration and continuous delivery (CI/CD) experience, and enterprise-grade security and governance.
+
+
+- Palette helps you unite the development and operations to a single platform.  This unification helps you achieve faster builds, delivery, and scaling of applications with credence.
+
+
+- The infrastructure has event-driven autoscaling and triggers that enable elastic provisioning for self-managed infrastructure.
+
+
+- Leverage extensive authentication and authorization capabilities by using Azure Active Directory and dynamic rules enforcement, across multiple clusters with Azure Policy.
+
+
+![An Azure AKS architecture diagram](clusters_azure_architecture_aks-diagram.png)
+
+
+# Azure Storage
+
+During an Azure cluster deployment, Palette creates an [Azure storage account](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-overview) and storage container. Palette copies the base virtual hard disk (VHD) image to the Palette default storage container in the default Palette storage account. The storage account Palette created has unrestricted access and has an auto-generated name. You can attach a custom storage account or storage containers to the Azure cluster. 
+
+Before the Azure cluster creation process, you must have created custom storage accounts or containers. All custom storage accounts and containers will be listed in the **Cluster config** page during the cluster creation process. If you need help creating a custom storage account or container, check out the Azure [Create a Storage Account](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) guide or the Azure [Manage Containers](https://learn.microsoft.com/en-us/azure/storage/blobs/blob-containers-portal) guide.
```

Comment:
All custom storage accounts and containers will be listed in the **Cluster config** page during the cluster creation process.

i guess the above sentence is in passive voice